afs/media/js/views/components/reports/project.js

```diff
@@ -41,6 +41,40 @@ define([
             self.documentationCards = {};
             self.summary = params.summary;
 
+            self.childProjectTableConfig = {
+                ...self.defaultTableConfig,
+                columns: Array(4).fill(null)
+            };
+
+            self.collectionOfChildProjects = ko.observableArray();
+            self.visible = {
+                childProjects: ko.observable(true),
+            };
+
+            const resourceId = ko.unwrap(self.reportMetadata).resourceinstanceid;
+            const relatedResources = async() => {
+                const result = await reportUtils.getRelatedResources(resourceId);
+                const rrs = result?.related_resources;
+                let childProjects =[];
+
+                const relationships = result?.resource_relationships;
+                for (indv_relationship in relationships) {
+                    if (relationships[indv_relationship]?.resourceinstancefrom_graphid ===
+                        relationships[indv_relationship]?.resourceinstanceto_graphid) {
+                            childProjects.push(relationships[indv_relationship]?.resourceinstanceidfrom);
+                    };
+                }
+
+                self.collectionOfChildProjects(rrs.filter(rr => 
+                    childProjects.includes(rr.resourceinstanceid)));
+                
+                for (child of self.collectionOfChildProjects()) {
```

Comment:
this is better done with array.map.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map